css/src/components/form/textarea.scss

```diff
@@ -87,3 +88,60 @@ $textarea-focus-box-shadow-size: 0 0.0625rem 0 0 !default;
 		}
 	}
 }
+
+$textarea-form-gap: 0.25em !default;
+$textarea-divider-radius: 0.5rem !default;
+
+.textarea-form {
+	display: grid;
+	gap: $textarea-form-gap;
+	padding-block: $control-padding-vertical;
+	padding-inline: $control-padding-horizontal;
+	transition: box-shadow 300ms linear;
+	border: $control-border-width solid transparent;
+	border-radius: $border-radius-lg;
+	border-color: $textarea-border-color;
+	outline-color: transparent;
```

Comment:
This `outline-color: transparent` is removing focus indicators altogether from the footer buttons. We should either remove this, or move it to more specifically target the textarea, or add focus indicators back in for the footer buttons.